java/src/jmri/jmrit/logixng/actions/swing/ActionEntryExitSwing.java

```diff
@@ -74,6 +74,12 @@ public boolean validate(@Nonnull List<String> errorMessages) {
         _selectNamedBeanSwing.validate(action.getSelectNamedBean(), errorMessages);
         _selectOperationSwing.validate(action.getSelectEnum(), errorMessages);
 
+        DestinationPoints dp = _selectNamedBeanSwing.getBean();
+        boolean isReverse = _selectOperationSwing.getEnum() == Operation.SetNXPairReversed;
+        if (dp != null && isReverse && dp.isUniDirection()) {
+            errorMessages.add(Bundle.getMessage("ActionEntryExit_SetReversedError", dp.getDisplayName()));
+        }
```

Comment:
Please put this code within an if statement:
```
if (_selectNamedBeanSwing.getAddressing() == NamedBeanAddressing.Direct
        && _selectOperationSwing.getAddressing() == NamedBeanAddressing.Direct) {
    ...
}
```
If any of these have indirect addressing, you don't know in advance whenether or not reversed is allowed.